src/test/java/net/masterthought/cucumber/ConfigurationTest.java

```diff
@@ -420,4 +421,58 @@ public void setNotFailingStatuses_SkipsNullValues() {
         // then
         assertThat(configuration.getNotFailingStatuses()).containsExactly(notFailingStatus);
     }
+
+    @Test
+    public void getOutputHandlers_shouldContainFilesystemOutputHandler() {
```

Comment:
Actually `getOutputHandlers_shouldContainFilesystemOutputHandler` does not contain, it returns